site/content/contribute/why-contribute/_index.md

```diff
@@ -0,0 +1,99 @@
+
+## Subsection 3 — You’d like to make something more inclusive or accessible
+
+1. Accessibility is one of the most overlooked yet most important features of modern software development, and we’re eager to improve on the accessibility of the features in our open source project and its documentation.
+2. Problems with the accessibility or the inclusivity of both features in the codebase and pieces of content describing Mattermost are bugs, so we treat them as such in our development process. See subsections 1 and 2 for the description on how you can submit your contribution.
+3. When you contribute a change that incorporates an adjustment based on the principles of accessibility and inclusivity, please use this guide to back you up in the PR, ticket, or post.
+
+## Subsection 4 — You have a feature idea
+
+1. Thank you for your enthusiasm! You can act on feature ideas in a few ways:
+    1. Take a look at our [product roadmap](https://mattermost.com/roadmap/). There’s a chance we might already be building the thing you want.
+    2. [Provide input on features](https://portal.productboard.com/mattermost/33-what-matters-to-you/tabs/117-potential-future-offerings) we’re considering to let us know what matters the most to you.
+    3. [Participate in a survey](https://portal.productboard.com/mattermost/33-what-matters-to-you/tabs/115-help-us-learn-more-through-surveys) to help us better understand how to meet the needs of our users.
+    4. Discuss your idea with our community in the [Feature Proposals channel](https://community.mattermost.com/core/channels/feature-ideas).
+    5. Build an app! Mattermost has a rich framework full of tools to help you add the features you want that don’t quite work as core additions to Mattermost.
+        1. [Webhooks]({{< ref "/integrate/webhooks/" >}}) are the easiest type of app to create and enable you to integrate with the vast majority of modern web tools.
+        2. [Slash commands]({{< ref "/integrate/slash-commands/" >}}) are apps that respond to triggers sent as messages inside Mattermost.
+        3. More complicated apps can be built as standard HTTP services in any programming language using a robust API. [Get started here]({{< ref "/integrate/apps/" >}}).
+        4. If you’re willing to deal with more complex development overhead, [plugins]({{< ref "/integrate/plugins/" >}}) enable you to alter every component of the Mattermost experience.
+
+## Subsection 5 — You’re looking to practice your skills or give back to the community
+
+1. We love developers who are passionate about open-source. If you’re looking to tackle an interesting problem, we’ve got you covered! Feel free to check out [the help wanted tickets on GitHub](https://github.com/search?utf8=%E2%9C%93&q=is%3Aopen+archived%3Afalse+org%3Amattermost+label%3A%22Help+Wanted%22++label%3A%22Up+For+Grabs%22). To take one on, just comment on the issue and follow the process outlined in article 1.1 of this guide.
```

Comment:
```suggestion
1. We love developers who are passionate about open-source. If you’re looking to tackle an interesting problem, we’ve got you covered! Feel free to check out [the help wanted tickets on GitHub](https://github.com/search?utf8=%E2%9C%93&q=is%3Aopen+archived%3Afalse+org%3Amattermost+label%3A%22Help+Wanted%22++label%3A%22Up+For+Grabs%22). To take one on, just comment on the issue and follow the process outlined in section 1, sub-section 1 of this guide.
```

Reply:
@neflyte - Can you please replace the section/subsection reference with a direct link to the section where readers can  see more details?